FAQ.md

```diff
@@ -0,0 +1,19 @@
+Q: When running pytest for a code challenge, I'm getting unexpected results or errors. What should I check?
+
+A: Before running pytest, it's essential to check your current directory (path) to ensure that you are in the correct location for the specific code challenge. Running pytest from an incorrect path may cause errors or unexpected results because the test files or modules may not be found. To verify your location, you can use the pwd command (on macOS/Linux) or cd and dir commands (on Windows) to navigate to the correct directory. Once confirmed, you can run pytest to test your code accurately.
+
+Q: When running pytest, I noticed that after specifying the filename, the rest of the command (like -p no:cacheprovider) isn’t necessary. Can I simplify the command? 
+
+A: Yes! When running pytest on a specific test file, you usually only need to provide the path to the file. For example, to test all_construct_test.py, you can simply run:
+
+pytest all-construct/all_construct_test.py
+
+The extra option -p no:cacheprovider is typically used to disable caching but is not required unless you have a specific reason to avoid pytest’s caching. In most cases, just specifying the filename is sufficient to run your tests.
```

Comment:
please note that caching could prevent code changes from being reflected in tests, so if people run `pytest` and their changes are not being tested, they will need to disable the cache to ensure their code is actually being tested.

Reply:
I was able to test it out and I was mistaken about the extra argument. Ive pulled references from the README file, so we can drop references here as well. Thanks for calling out that we don't need it!